package.json

```diff
@@ -8,7 +8,8 @@
     "build": "node scripts/build/build.js",
     "commit": "git show --format=\"%h\" --no-patch",
     "db": "ts-node scripts/migrations/db.ts",
-    "lint": "eslint --ext .ts,.tsx . --max-warnings 420",
+    "lint": "eslint --max-warnings 2500",
```

Comment:
Is this the current warnings?
I think we should be working towards reducing the lint warnings so I normally like to cap it at the current amount and start fixing and reducing as much as I can over time

Reply:
I think it's ~1300 but it's because I added some new rules.

I think during our refactoring we shouldn't care about that but I think the rules are important so it highlights the "wrong patterns"